Source/Microsoft.Teams.Apps.RemoteSupport.Common/Constants.cs

```diff
@@ -146,5 +146,10 @@ public static class Constants
         /// Closed requests command id in the manifest file.
         /// </summary>
         public const string ClosedCommandId = "closedrequests";
+
+        /// <summary>
+        /// Text block card id for first observed on text in remote support request card
+        /// </summary>
+        public const string IssueOccurredOnId = "IssueOccurredOn";
```

Comment:
Instead of putting this into a generic Constants class, please move it to another class for card template field IDs.
Include "DateValidationMessage" in that, to be consistent.

Reply:
make sense, moved it to a new class CardConstants and added const which are related to cards and remote support project and do not have use in common library.